src/core/annotation.js

```diff
@@ -987,6 +987,22 @@ class WidgetAnnotation extends Annotation {
     const vPadding = defaultPadding + Math.abs(descent) * fontSize;
     const defaultAppearance = this.data.defaultAppearance;
 
+    if (this.data.comb) {
+      const combWidth = (totalWidth / this.data.maxLen).toFixed(2);
+      let buf = `/Tx BMC q BT ${defaultAppearance} 1 0 0 1 ${hPadding} ${vPadding} Tm`;
+      let first = true;
+      for (const character of value) {
+        if (first) {
+          buf += ` (${character}) Tj`;
```

Comment:
To reduce the number of intermediate strings being created, we normally don't append strings to strings. Instead, we push all strings to an array and join them at the end, which is a bit more efficient. Can we do that here as well? Look at https://github.com/mozilla/pdf.js/blob/50bc4a18e8c564753365d927d5ec6a6d2cce3072/src/core/core_utils.js#L113-L131 for the pattern we use.

Moreover, this will also fix the line `let buf = ...` above since that is longer than the 80 character limit. If we split that large string into two strings we can simply push those into the array so it fits with the character limit again (I'm surprised ESLint doesn't complain about this already...).